index.markdown

```diff
@@ -482,6 +482,18 @@ assumptions about your attendees. Get someone who is demographically very
 different from you to check your marketing material through to see if it makes
 sense and isn't offensive to someone who doesn't share your background.
 
+Many event attendees expect events to have and enforce a code of conduct. Having
+such a code encourages diversity of participation and creates and maintains a
+welcoming envrionment. It is highly recommended to adopt a code of conduct for
+your event such as the [Hack Code of Conduct](http://hackcodeofconduct.org),
+which may be either referenced directly or by creating a custom version of this
```

Comment:
I'd break this sentence up, it is pretty long.

It is therefore highly recommended to adopt a code of conduct for your event. One such example is the [Hack Code of Conduct](http://hackcodeofconduct.org), but there are several other appropriate examples. You can either reference your chosen code of conduct direct, or create your own customised version. Make sure to communicate clearly to attendees ahead of time the existence of this code and the enforcement policy behind it.